### PR TITLE
fix(deps): update opi-api digest and api changes

### PR DESCRIPTION
**Pull request by sandersms**

### CODE REVIEW

File: pkg/backend/aio_test.go [hunk 1]
```diff
@@ -479,15 +479,16 @@ func TestBackEnd_ListAioVolumes(t *testing.T) {
...
-		"no required field": {
-			in:      "",
-			out:     []*pb.AioVolume{},
-			spdk:    []string{},
-			errCode: codes.Unknown,
-			errMsg:  "missing required field: parent",
-			size:    0,
-			token:   "",
-		},
+		// Parent field not in message - depricate test
```
Comment (artek-koltun) [inline pkg/backend/aio_test.go:482]: we can just remove this test since there is no parent anymore

File: pkg/frontend/blk_test.go [hunk 2]
```diff
@@ -426,15 +426,16 @@ func TestFrontEnd_ListVirtioBlks(t *testing.T) {
...
-		"no required field": {
-			"",
-			[]*pb.VirtioBlk{},
-			[]string{},
-			codes.Unknown,
-			"missing required field: parent",
-			0,
-			"",
-		},
+		// Parent field not in message - depricate test
```
Comment (artek-koltun) [inline pkg/frontend/blk_test.go:429]: we can remove this test

File: pkg/backend/aio_test.go (hunk 1 above)
Comment (sandersms) [reply pkg/backend/aio_test.go:482]: Great - I deprecated it to start with but will remove it completely.

**Review by artek-koltun — COMMENTED** [review]
Ok, I checked this patch thoroughly. We need to update some more lines:
```diff
diff --git a/pkg/backend/nvme_path.go b/pkg/backend/nvme_path.go
index e17f870..4aafceb 100644
--- a/pkg/backend/nvme_path.go
+++ b/pkg/backend/nvme_path.go
@@ -273,7 +273,7 @@ func (s *Server) StatsNvmePath(ctx context.Context, in *pb.StatsNvmePathRequest)
 }
 
 func (s *Server) opiTransportToSpdk(transport pb.NvmeTransportType) string {
-       return strings.ReplaceAll(transport.String(), "NVME_TRANSPORT_", "")
+       return strings.ReplaceAll(transport.String(), "NVME_TRANSPORT_TYPE_", "")
 }
 
 func (s *Server) opiMultipathToSpdk(multipath pb.NvmeMultipath) string {
diff --git a/pkg/utils/server.go b/pkg/utils/server.go
index b017426..96e4089 100644
--- a/pkg/utils/server.go
+++ b/pkg/utils/server.go
@@ -148,5 +148,5 @@ func OpiAdressFamilyToSpdk(adrfam pb.NvmeAddressFamily) string {
                return ""
        }
 
-       return strings.ReplaceAll(adrfam.String(), "NVME_ADRFAM_", "")
+       return strings.ReplaceAll(adrfam.String(), "NVME_ADDRESS_FAMILY_", "")
 }
```
and update `godpu` sha256 [here](https://github.com/opiproject/opi-spdk-bridge/blob/457e5ebd5e030db353a4fecb3052b7dea1c63d19/docker-compose.yml#L116) to `4301eadbb4bd4803eb993a9fd40ae52f132e1c5489996af60777ab320765a8e0`
sha256 was taken from [here](https://github.com/opiproject/godpu/pkgs/container/godpu/169469567?tag=main) and the CI should pass

**Comment by sandersms** [comment]
> Ok, I checked this patch thoroughly. We need to update some more lines:
> 
> ```diff
> diff --git a/pkg/backend/nvme_path.go b/pkg/backend/nvme_path.go
> index e17f870..4aafceb 100644
> --- a/pkg/backend/nvme_path.go
> +++ b/pkg/backend/nvme_path.go
> @@ -273,7 +273,7 @@ func (s *Server) StatsNvmePath(ctx context.Context, in *pb.StatsNvmePathRequest)
>  }
>  
>  func (s *Server) opiTransportToSpdk(transport pb.NvmeTransportType) string {
> -       return strings.ReplaceAll(transport.String(), "NVME_TRANSPORT_", "")
> +       return strings.ReplaceAll(transport.String(), "NVME_TRANSPORT_TYPE_", "")
>  }
>  
>  func (s *Server) opiMultipathToSpdk(multipath pb.NvmeMultipath) string {
> diff --git a/pkg/utils/server.go b/pkg/utils/server.go
> index b017426..96e4089 100644
> --- a/pkg/utils/server.go
> +++ b/pkg/utils/server.go
> @@ -148,5 +148,5 @@ func OpiAdressFamilyToSpdk(adrfam pb.NvmeAddressFamily) string {
>                 return ""
>         }
>  
> -       return strings.ReplaceAll(adrfam.String(), "NVME_ADRFAM_", "")
> +       return strings.ReplaceAll(adrfam.String(), "NVME_ADDRESS_FAMILY_", "")
>  }
> ```
> 
> and update `godpu` sha256 [here](https://github.com/opiproject/opi-spdk-bridge/blob/457e5ebd5e030db353a4fecb3052b7dea1c63d19/docker-compose.yml#L116) to `4301eadbb4bd4803eb993a9fd40ae52f132e1c5489996af60777ab320765a8e0` sha256 was taken from [here](https://github.com/opiproject/godpu/pkgs/container/godpu/169469567?tag=main) and the CI should pass

I also found a few updates needed in the README file and scripts for the same enums with an extensive search on the entire folder.  I added those changes to the PR also.

**Review by artek-koltun — APPROVED** [review]
LGTM

**Pull request merged**